content/conditions/cold/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "layout": "content-simple",
+  "title": "Common cold",
+  "nonEmergencyCallout": true,
+  "choicesOrigin": "conditions/cold-common/pages/introduction.aspx",
+  "content": {
+    "header": [
+      {
+        "type": "markdown",
+        "value": "You can often treat a cold without seeing your GP. You should begin to feel better in about a week or two."
+      }
+    ],
+    "main": [
+      {
+        "type": "panel",
+        "value": {
+          "main": "!file=panel-1-main.md",
+          "footer": "The symptoms are the same in adults and children. Sometimes, symptoms last longer in children."
+        }
+      },
+      {
+        "type": "markdown",
+        "value": "!file=main-content-1.md"
+      },
+      {
+        "type": "panel",
+        "value": {
+          "main": "!file=panel-2-main.md",
+          "footer": "[Find services that provide urgent care in your area](http://www.nhs.uk/service-search/Minor-injuries-unit/LocationSearch/551)"
```

Comment:
This doesn't feel like the correct place to link to. Might be worth double checking with content